### ISSUE TITLE
Reproducing the training results

### ISSUE THREAD

**Issue by MarkoKostiv** [liebharc/homr#51]
I’m working on improving the articulation predictions and fine-tuning the decoder branch. However, I’m currently having trouble reproducing the results reported for run 242:

- In the [Training.md, run 242](https://github.com/liebharc/homr/blob/main/Training.md#run-242), the reported SER is 23%
- The exported ONNX model for run 242 produces an SER of 20%
- Training model from scratch for 62,856 steps (with the default git config) produces SER 111%

Could you provide any insights on the training? The loss curves suggest that eval loss still decreases, will training longer do the trick?

Additionally, if you could release the pre-trained PyTorch weights, that would be greatly appreciated.

Thank you!

**Comment by liebharc** [comment]
Ah yes that makes sense, you find all the pytorch weights here: https://github.com/liebharc/homr/releases/tag/checkpoints

**Comment by liebharc** [comment]
I've updated #52 , training on the previous version of the model worked in this case. Although it's a special case as only the lift branch of the decoder was trained. I don't know the details of what you want to achieve. But to improve the overall performance of the articulation detection, you likely need to train the encoder and decoder. So the freezing of parameters which I used would not be recommended in your case.

If you happen to do a full run on the code level of the PR, then I would be happy if you would share the resulting pytorch model. I would be interested in how the full training from scratch compares to the two-step training approach which I just used.

**Comment by liebharc** [comment]
With #84 the training should be more reliable now and multiple people reported successful training runs. I will go ahead then and close this issue.
